examples/cra-dapp/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "cra-dapp",
+  "type": "module",
```

Comment:
we need this change ?

Reply:
looks orthogonal to this PR

Reply:
We don't really need it, but it fixes a warning when running our example dapp.

<img width="432" alt="Screenshot 2024-03-29 at 19 40 09" src="https://github.com/FuelLabs/fuels-wallet/assets/7074983/60f4c5d7-7800-4206-9057-0cf2307c54b8">

That link recommends that we set up ESM as default in our `package.json`.
Reference: https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated

If you don't mind we can move forward with this change.